src/Reflex/Spider/Internal.hs

```diff
@@ -111,46 +113,35 @@ import Reflex.PerformEvent.Base (PerformEventT)
 #ifdef DEBUG_TRACE_EVENTS
 import qualified Data.ByteString.Char8 as BS8
 import System.IO (stderr)
-#endif
-
-#ifdef DEBUG_TRACE_EVENTS
+import Data.List (isPrefixOf)
 
 withStackOneLine :: (BS8.ByteString -> a) -> a
```

Comment:
probably would be better not to have the same ifdef spanning both imports and definitions, although that's not new to this PR